src/main/java/org/usfirst/frc4904/robot/commands/Winch.java

```diff
@@ -0,0 +1,34 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+
+public class Winch extends MotorConstant {
+  public static final double DEFAULT_WINCH_SPEED = 0; // TODO: This is an untested value
+  protected Motor winchMotor;
+  protected double speed;
+
+  /**
+   * Winch the robot to hang on the generator switch
+   * 
+   * @param winchMotor The winch motor
+   * @param speed      The speed to winch up with
+   */
+  Winch(Motor winchMotor, double speed) {
+    super("Winch", winchMotor, speed);
+  }
```

Comment:
we're probably always going to set the same winch motor to the same speed, so there only needs to be one constructor that takes in no args that just calls `super`. Same with the above classes too